pkgs/development/ocaml-modules/janestreet/janePackage.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, ocaml, jbuilder, findlib }:
+
+{ name, version ? "0.9.0", buildInputs ? [], hash, meta, ...}@args:
```

Comment:
@mt-caret If you change all the package versions at once you should start by changing the default argument here.